mesmer/core/auto_regression.py

```diff
@@ -2,6 +2,76 @@
 import xarray as xr
 
 
+def _select_ar_order_xr(data, dim, maxlag, ic="bic"):
```

Comment:
```suggestion
def select_ar_order(data, dim, maxlag, ic="bic"):
```

Maybe rename this to be the "top level function"?